scripts/ci/ci_flake8.sh

```diff
@@ -50,6 +50,8 @@ function run_flake8() {
     fi
 }
 
+get_ci_environment
```

Comment:
Can this be called from prepare_ci_build? It seems everywhere we call one we call the other.

Reply:
Not really. There is one place that between get_ci_environment and  prepare_ci_build we check if this is a "cron" job (We find it out in get_ci_environment)  and if it is, we override few variables to make the build from scratch.

Reply:
```
    get_ci_environment

    # In case of CRON jobs we run builds without cache and upgrade to latest requirements
    if [[ "${CI_EVENT_TYPE:=}" == "schedule" ]]; then
        echo
        echo "Disabling cache for scheduled jobs"
        echo
        export DOCKER_CACHE="no-cache"
        export PULL_BASE_IMAGES="true"
        export UPGRADE_TO_LATEST_REQUIREMENTS="true"
    fi

```
    prepare_ci_build